agent/exploits/cve_2024_9487.py

```diff
@@ -0,0 +1,101 @@
+import datetime
+import re
+from urllib import parse as urlparse
+
+import requests
+from packaging import version
```

Comment:
Why not semver??

Reply:
what do you mean?

Reply:
semver refers to a package that handles semantic versions. The version you are using are semantic version. Mouhcine comment is valid.